### PR TITLE
Update copyrights and licenses

### PR DESCRIPTION
**Pull request by LeonidBeynenson**

### CODE REVIEW

File: mmseg/models/utils/channel_shuffle.py [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# Copyright (C) 2021 Intel Corporation
```
Comment (evgeny-izutov) [inline mmseg/models/utils/channel_shuffle.py:4]: It's part of Lite-HRNet repo.

File: mmseg/models/backbones/icnet.py [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# Copyright (C) 2021 Intel Corporation
```
Comment (evgeny-izutov) [inline mmseg/models/backbones/icnet.py:1]: Original repo: https://github.com/hszhao/ICNet

File: mmseg/models/backbones/stdcnet.py [hunk 3]
```diff
@@ -1,3 +1,7 @@
+# Copyright (C) 2021 Intel Corporation
```
Comment (evgeny-izutov) [inline mmseg/models/backbones/stdcnet.py:4]: The original repo: https://github.com/MichaelFan01/STDC-Seg

File: mmseg/models/decode_heads/hamburger_head.py [hunk 4]
```diff
@@ -1,3 +1,7 @@
+# Copyright (C) 2021 Intel Corporation
```
Comment (evgeny-izutov) [inline mmseg/models/decode_heads/hamburger_head.py:1]: The original repo: https://github.com/Gsunshine/Enjoy-Hamburger

File: mmseg/models/decode_heads/ic_head.py [hunk 5]
```diff
@@ -1,3 +1,7 @@
+# Copyright (C) 2021 Intel Corporation
```
Comment (evgeny-izutov) [inline mmseg/models/decode_heads/ic_head.py:1]: The original repo: https://github.com/hszhao/ICNet

**Review by evgeny-izutov — CHANGES_REQUESTED** [review]

File: mmseg/models/backbones/icnet.py (hunk 2 above)
Comment (LeonidBeynenson) [reply mmseg/models/backbones/icnet.py:1]: Ok

File: mmseg/models/backbones/stdcnet.py (hunk 3 above)
Comment (LeonidBeynenson) [reply mmseg/models/backbones/stdcnet.py:4]: Ok

File: mmseg/models/decode_heads/hamburger_head.py (hunk 4 above)
Comment (LeonidBeynenson) [reply mmseg/models/decode_heads/hamburger_head.py:1]: Ok

File: mmseg/models/decode_heads/ic_head.py (hunk 5 above)
Comment (LeonidBeynenson) [reply mmseg/models/decode_heads/ic_head.py:1]: Ok

**Review by evgeny-izutov — APPROVED** [review]

**Pull request merged**